Firestore/third_party/abseil-cpp/absl/numeric/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+#
```

Comment:
This should be done in a separate CL, based of the `abseil-mirror` branch and then merged to `master`. This way we have pristine sources and we can track how we got them. Also we should be importing from the same version.

Reply:
> Also we should be importing from the same version.

How can I find out the exact Abseil version to use?

Reply:
This no longer applies. Gil, thanks for helping with Abseil.